apps/admin/src/app/(api)/api/v1/token/route.ts

```diff
@@ -40,13 +40,26 @@ export const POST = async (req: NextRequest) => {
             }
 
             if ((sensorData.needsScript || data.needScript) && sensorData.script) {
+                const additionalData: {
```

Comment:
```suggestion
                let additionalData: {
```
und dann vors if ziehen